index.js

```diff
@@ -24,67 +30,72 @@ function getRandomNumber(min, max) {
 
 function checkGuess() {
   // Get value from guess input element
-  const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
+  let guess = Number(guessInput.value, 10);
+ 
 
   hideAllMessages();
 
+
   if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
+    numberOfGuessesMessage.style.display = "";
     numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
```

Comment:
Here you could check the number of guesses and have it output the proper grammar based on the amount of guesses. Just a quick example:

`numberOfGuessesMessage.innerHTML = `You made ${attempts} guess${attempts <= 1 ? '' : 'es'}`;`